src/modules/datepicker/directives/datepicker.directive.ts

```diff
@@ -60,6 +74,9 @@ export class SuiDatepickerDirective
                 this.config = new TimeConfig();
                 break;
         }
+        if (this._initialView != undefined) {
```

Comment:
Do you mean a parameter on `CalendarMappings`? so it can be used like
```
export class DateMappings extends CalendarMappings {
    constructor(initialView?:CalendarViewType) {
        super(initialView || CalendarViewType.Date);
```